tests/conftest.py

```diff
@@ -2976,6 +3017,71 @@ def finalizer():
     return factory
 
 
+@pytest.fixture()
+def multi_snapshot_restore_factory(snapshot_restore_factory):
+    """
+    Snapshot restore factory. Calling this fixture creates set of new PVC out of the
+    specified VolumeSnapshot.
```

Comment:
Please improve this statement. This gives an impression that multiple PVCs will be created from the same VolumeSnapshot.